lib/recaptcha/token.rb

```diff
@@ -0,0 +1,19 @@
+module Recaptcha
+  module Token
+    require 'json'
+
+    def self.secure_token
```

Comment:
since it's a public api now, a basic test like `it 'generates a token'` would be nice